content/nginx-one/glossary.md

```diff
@@ -15,9 +15,11 @@ This glossary defines terms used in the F5 NGINX One Console and F5 Distributed
 | Term        | Definition |
 |-------------|-------------|
 | **Config Sync Group** | A group of NGINX systems (or instances) with identical configurations. They may also share the same certificates. However, the instances in a Config Sync Group could belong to different systems and even different clusters. For more information, see this explanation of [Important considerations]({{< ref "/nginx-one/how-to/config-sync-groups/manage-config-sync-groups.md#important-considerations" >}}) |
+| **Control Plane** | The control plane is the part of a network architecture that manages and controls the flow or data or traffic (the Data Plane). It is responsible for system-level tasks such as routing and traffic management. |
 | **Data Plane** | The data plane is the part of a network architecture that carries user traffic. It handles tasks like forwarding data packets between devices and managing network communication. In the context of NGINX, the data plane is responsible for tasks such as load balancing, caching, and serving web content. |
 | **Instance** | An instance is an individual system with NGINX installed. You can group the instances of your choice in a Config Sync Group. When you add an instance to NGINX One, you need to use a data plane key. |
 | **Namespace** | In F5 Distributed Cloud, a namespace groups a tenant’s configuration objects, similar to administrative domains. Every object in a namespace must have a unique name, and each namespace must be unique to its tenant. This setup ensures isolation, preventing cross-referencing of objects between namespaces. You'll see the namespace in the NGINX One Console URL as `/namespaces/<namespace name>/` |
+| **NGINX Agent**                      | A lightweight software component installed on NGINX instances to enable communication with the NGINX One console.                                      |
```

Comment:
Would it make sense to link this to the top level of where you're placing NGINX Agent docs? (I'm guessing `{{< ref "/nginx-one/agent" >}}` )

Reply:
I would hold on that as there is an active effort to integrate all the content into the N1 docs and the section will "disolve"

Reply:
Withdrawing. This is a WIP.